### PR TITLE
added extra polarizations antenna pattern functions

### PR DESCRIPTION
**Pull request by srashtig**
Modified 'antenna_pattern' function to include other polarizations(vector - fx, fy and scalar - fb, fi) antenna pattern functions.

### CODE REVIEW

**Comment by ahnitz** [comment]
@srashtig Can you rebase your branch from the current master and take care not to delete the methods which your branch doesn't currently have? At the moment it seems as though you have accidentally deleted those changes.

**Comment by srashtig** [comment]
Hi Alex, I have created a new pull request altogether over [here](https://github.com/gwastro/pycbc/pull/3486). Perhaps this can be closed.

File: pycbc/detector.py [hunk 1]
```diff
@@ -240,26 +274,44 @@ def time_delay_from_detector(self, other_detector, right_ascension,
...
         polarizations.
+
         """
-        h_lal = lalsimulation.SimDetectorStrainREAL8TimeSeries(
-                hp.astype(np.float64).lal(), hc.astype(np.float64).lal(),
-                longitude, latitude, polarization, self.frDetector)
-        return TimeSeries(
-                h_lal.data.data, delta_t=h_lal.deltaT, epoch=h_lal.epoch,
-                dtype=np.float64, copy=False)
+        print pol_flag
```
Comment (ahnitz) [inline pycbc/detector.py:285]: These statements don't work in python3 and seem to be left in from debugging.

**Review by ahnitz — APPROVED** [review]

File: pycbc/detector.py (hunk 1 above)
Comment (ahnitz) [reply pycbc/detector.py:285]: Woops sorry, looking at the wrong PR, I'll close this one.

**Pull request closed without merging**